hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCreateHandle.java

```diff
@@ -136,6 +136,7 @@ public WriteStatus close() {
       stat.setPaths(new Path(config.getBasePath()), path, tempPath);
       stat.setTotalWriteBytes(FSUtils.getFileSize(fs, getStorageWriterPath()));
       stat.setTotalWriteErrors(status.getFailedRecords().size());
+      stat.setTotalTimeTakenForInsert(timer.endTimer());
```

Comment:
lets rename these time variables based on the handle type i.e `setTotalCreateTime()`

Reply:
done